meetings/2022-12-14.md

```diff
@@ -55,11 +70,21 @@
   * https://github.com/solid/webid-profile/issues/55 ( https://github.com/solid/webid-profile/pull/80 )
 * SC: Involving the DEIT (Diversity, Equity, Inclusion Team) to review UCs:
   * https://github.com/solid/deit/blob/main/meetings/2021-11-09.md#deit-to-review-use-cases-in-specification-development
-
-
-### Documentation and Discovery
-* SC: We have a lot of information and so there is room to improve UCs discovery and communication.
-* SC: Questions/Concerns on how to improve:
+* WT: Two small questions, one is the user stories issues list, who manages that? Who puts on the labels and so on?
+* SC: Anyone can submit a user story. It's for the organization of it, the tagging, I'd have to double-check the access controls, should be open to anyone in the community. If issues labeling, I can circle back with you.
+* TBL: I suppose ... high level roles .. for solid in general, ... they want to, not just produce a list at one level, but recognize some are, I should be able to like somebody's stuff, high level, social media to collaborate, why we are doing all this, ... point to high level one that when you try to match them up to low level ... encourages to long term ...???
+* SC: You are cutting out a bit. Those who have high level view on user stories, some people who tag it initially, but people need to have awareness of all the user stories so they can manage it that way.  Maybe this is something we can follow up on. The labels, most do not have a description, but we could use better descriptions. Some are clear enough. 
+* TBL: We also have a roadmap we could tie this to, things like media.
+* WT: Thanks Sarven and Tim. Useful to have a more clear understanding of how or if the structure of the user stories, so we don't expect everyone to go through them all, but some indication this might be useful to me cause I'm working on X or Y.
```

Comment:
```suggestion
* WT: Thanks Sarven and Tim. Useful to have a more clear understanding of the structure of the user stories, so we don't expect everyone to go through them all, but some indication "this might be useful to me cause I'm working on X or Y."
```